lib/api/controllers/securityController.js

```diff
@@ -55,6 +55,144 @@ const
 
 /**
  * @param {Kuzzle} kuzzle
+ * @param {Request} request
+ * @param {object} pojoUser
+ */
+const createGenerator = Bluebird.coroutine(function* persistUserAndCredentials (kuzzle, request, pojoUser) {
```

Comment:
nitpicking: `createGenerator` is not a generator,  `persistUserAndCredentials` is.